### ISSUE TITLE
Skip the building of OpenCV once it has already been built or allow path specification

### ISSUE THREAD

**Issue by adishavit** [biggzlar/opencv_js_compiler#2]
Basically, CMake already knows how to do this.

Which brings up a more interesting question, why use Python at all and not CMake?
Yes, it may be easier to script, but it is basically just setting up the CMake arguments and putting stuff together like CMake does.

**Comment by biggzlar** [comment]
Absolutely, python was a quick and dirty way to make everything happen fast. I don't have a lot of experience with cmake so I'm not sure how to drop python and still have everything work easily.

**Comment by adishavit** [comment]
I suspect the real way to do it would be to modify OpenCV's CMake file itself for emscripten. 

Let keep working with Python and leave this open for later.  

Though, avoiding the rebuild would be great. 
Probably 2 scripts: One for OpenCV the other for the "project".

**Comment by biggzlar** [comment]
Sounds good, also thanks for your insights so far!
